application/plugins/sms_to_twitter/config/sms_to_twitter.php

```diff
@@ -12,3 +12,7 @@
 */
 // phpcs:enable
 $config['twitter_code'] = 'TW';
+
+// Fill with your OAuth credentials
+$config['consumer_key'] = '';
```

Comment:
Populate default with existing keys maybe?

Reply:
We should update this in release notes in that case, that people who wants this plugin needs to generate their own keys in this case.

Reply:
Maybe I don't understand what these keys are for. Is it normal that all the world can send to this twitter account? I thought these are private. What would be the purpose of such credentials if it is world writable?

Reply:
Couldn't find docs of this type of keys on twitter, probably this plugin needs to be updated with the newer twitter api's.